source/MapDetailPanel.cpp

```diff
@@ -572,8 +572,8 @@ void MapDetailPanel::DrawKey()
 	{
 		// Each system is colored by the number of outfits for sale.
 		static const string LABEL[2][4] = {
-			{"None", "1", "5", "10+"},
-			{"None", "1", "30", "60+"}};
+			{"10+", "5", "1", "None"},
+			{"60+", "30", "1", "None"}};
 		static const double VALUE[4] = {-1., 0., .5, 1.};
```

Comment:
Probably just reverse this to have the colours match the new order of the labels.
```suggestion
		static const double VALUE[4] = {1., .5, 0., -1.};
```